app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -32,7 +37,24 @@ def host_sla(host)
       end
 
       def host_products(host)
-        host_subscription_facet(host)&.installed_products
+        host.content_facet&.installed_products
```

Comment:
This is raising an error: `undefined method `installed_products' for #<Katello::Host::ContentFacet:0x0000000006ccfd20>`

What was the reason for changing this method?

Reply:
Bad improvement :/